spec/controllers/curator/filestreams/file_sets_controller_spec.rb

```diff
@@ -30,6 +36,10 @@
         if can_be_exemplary?(file_set_type)
           attributes[:exemplary_image_of] = [{ ark_id: parent_obj.ark_id, _destroy: '1' }, { ark_id: parent_col.ark_id }]
         end
+        if has_image_thumbnail_300?(file_set_type)
+          file_attributes = load_json_fixture('image_file', 'files')
+          file_attributes[0]['metadata']['ingest_filepath'] = file_fixture('image_thumbnail_300.jpg').to_s
+        end
```

Comment:
Maybe I'm missing something, but is `file_attributes` used anywhere? There is no `attributes[:files] = file_attributes` like in the `let(:valid_attributes)` block above.